{"name": "Login", "status": "broken", "statusDetails": {"message": "TypeError: 'str' object is not callable\n", "trace": "  File \"C:\\Users\\Vigne\\PycharmProjects\\Behave_Feb23\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\Behave_Feb23\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps\\StepDefn.py\", line 30, in the_user_enters_valid_credentials\n    lp.password_textbox_id(\"secret_sauce\")\n"}, "steps": [{"name": "Given the user launches the application", "status": "passed", "start": 1676090123072, "stop": 1676090123589}, {"name": "When the user verifies the logo", "status": "passed", "start": 1676090123590, "stop": 1676090123629}, {"name": "Then the user enters valid credentials", "status": "broken", "statusDetails": {"message": "TypeError: 'str' object is not callable\n", "trace": "  File \"C:\\Users\\Vigne\\PycharmProjects\\Behave_Feb23\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Vigne\\PycharmProjects\\Behave_Feb23\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps\\StepDefn.py\", line 30, in the_user_enters_valid_credentials\n    lp.password_textbox_id(\"secret_sauce\")\n"}, "start": 1676090123629, "stop": 1676090123737}], "start": 1676090123070, "stop": 1676090123738, "uuid": "0a1b95b5-fa49-42a4-b453-e4bd7d5e0d1c", "historyId": "8fa95ae9eb30e067568b09ee12827b5a", "fullName": "Add to Cart: Login", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Add to Cart"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}